src/navigation/types.ts

```diff
@@ -1,23 +1,26 @@
-import { NativeStackScreenProps } from '@react-navigation/native-stack';
+export type { NativeStackScreenProps } from '@react-navigation/native-stack';
 
-export type { NativeStackScreenProps };
+const AuthStackScreens = {
+  Welcome: 'Welcome',
+  SignIn: 'SignIn',
+  SignUp: 'SignUp',
+} as const;
 
-export enum AppScreens {
-  // Auth
-  Welcome = 'Welcome',
-  // Main
-  Home = 'Home',
-}
+const MainStackScreens = {
+  Home: 'Home',
+} as const;
 
 export enum Stacks {
   AuthStack = 'AuthStack',
   MainStack = 'MainStack',
 }
 
-// TODO: Example -> [AppScreens.Welcome]: { exampleParam: string };
```

Comment:
Maybe I would leave an example for this new structure to let people know that the undefined is where you can pass the parameters